getting_started.ipynb

```diff
@@ -0,0 +1,2044 @@
+    "\n",
+    "- `timeout` specifies the maximum amount of time (in seconds) to wait for a response from the API before timing out. An     `APITimeoutError` will be raised in the case of a timeout.\n",
+    "\n",
+    "- `max_retries` sets the number of times the API request should be retried in case of retriable failure before giving up.\n",
+    "\n",
+    "- `model` specifies the model to be used.\n",
+    "\n",
+    "- `model_version` indicates the specific version of the chosen model to use.\n",
+    "\n",
+    "- See the [**API Reference**](https://api.python.langchain.com/en/latest/chat_models/langchain_openai.chat_models.azure.AzureChatOpenAI.html) for more details.\n",
+    "\n",
+    "- Other parameters may be available in different SDK's."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9edabc3",
+   "metadata": {},
+   "source": [
+    "### Wait in between API calls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "239cdb0b",
+   "metadata": {},
+   "source": [
+    "The number of API requests a user can make depends on their Azure plan and account settings. If too many requests are sent in a short period, an error may occur, prompting the user to wait for **x** amount of time before sending another request.\n",
+    "\n",
+    "When creating a model, one of the key parameters is the `max_retries` setting. The underlying Python OpenAI library will automatically wait and retry the call on your behalf at least 2 times by default before raising a `RateLimitError`. This behavior can be adjusted by setting a different value for `max_retries`.\n",
+    "\n",
+    "Visit the [**quotas and limits**](https://learn.microsoft.com/azure/ai-services/openai/quotas-limits) page to view detailed information related to account limits and restrictions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28a397b3-27b7-49f0-bccf-108f39196ee6",
+   "metadata": {},
+   "source": [
+    "## Model Usage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "7600d5eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.messages import HumanMessage, SystemMessage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a152eab-52e5-48f3-bc8f-0b410d65c7d0",
+   "metadata": {},
+   "source": [
+    "#### Using Messages from the `langchain_core.messages` Library\n",
+    "\n",
+    "The `langchain_core.messages` library allows the user to define messages for the model and assign roles to each message."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "259a462a-4a83-4f17-a337-be04d67d3020",
+   "metadata": {},
+   "source": [
+    "- LangChain-compatible chat models take a list of `messages` as `input` and return the AI message as `output`.\n",
+    "\n",
+    "- All messages have `role` and `content` properties. In the sample below, the roles are set by using the `SystemMessage` and `HumanMessage` classes. [**We'll cover more on this later**](#assigning-roles) .\n",
```

Comment:
For the anchor link, I think you'll need to include the full header for it to resolve

```suggestion
    "- All messages have `role` and `content` properties. In the sample below, the roles are set by using the `SystemMessage` and `HumanMessage` classes. [**We'll cover more on this later**](#Assigning-Roles-Using-LangChain-Messages) .\n",
```